README.md

```diff
@@ -578,7 +578,7 @@ If you want to build your own delivery method to deliver notifications to a spec
 This will generate a new `ApplicationDeliveryMethod` and `DeliveryMethods::Discord` class inside the `app/notifiers/delivery_methods` folder, which can be used to deliver notifications to Discord.
 
 ```ruby
-class DeliveryMethods::Discord < ApplicationDeliveryMethod
```

Comment:
This is the correct class. The generator will copy over `ApplicationDeliveryMethod` to your app.

https://github.com/excid3/noticed/blob/main/lib/generators/noticed/templates/application_delivery_method.rb.tt

Reply:
No worries! And that'll work the same, but I do want to encourage people use the generator.